Update access request enterprise description (#22284)
gravitational/teleport: docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx

| Teleport Enterprise Feature | Description |
| - | - |
| [Single Sign-On (SSO)](#sso) | Allows Teleport to integrate with existing enterprise identity systems. Examples include Active Directory, GitHub, Google Apps and numerous identity middleware solutions like Auth0, Okta, and so on. Teleport supports SAML and OAuth/OpenID Connect protocols to interact with them. |

> | [Access Requests](../../access-controls/access-requests.mdx) | Request access to specific resources and a user interface for teams to create and review requests to access infrastructure with escalated privileges. |

Reviewer: ```suggestion
| [Access Requests](../../access-controls/access-requests.mdx) | Request elevated access to roles or specific resources. |
```

## Access Requests

With Teleport, users can request access roles or to a specific set of resources.
Requests can be published to tools like Slack, Jira, and PagerDuty for easy integration with your organization's workflows.

> requesting specific resources such as a production server to allow fine-grain access for a targeted task.  The web interface

Reviewer 2: > requesting specific resources such as a production server to allow fine-grain access for a targeted task.  

Looks like the first half of this sentence was cut off; there's no subject. Also, extra spacing at the end.

Author: Please see updates

## Access Requests

With Teleport, users can request access to roles or to a specific set of resources
such as a production server. Requests can be published to tools like Slack, Jira,

> and PagerDuty for easy integration with your organization's workflows. The web 
> interface provides for creating and approving access requests that is only 
> available in Teleport Enterprise.

Reviewer 2: ```suggestion
and PagerDuty for easy integration with your organization's workflows. Teleport
Enterprise's web interface also provides for creating and approving access requests.
```

## Access Requests

> With Teleport users can request access to specific resources and roles with requests publishable to tools such as Slack, Jira, PagerDuty that fits to your organization's workflows. Authorized users and automated processes can approve the requests. The commercial edition of Access Requests allows for

Reviewer: ```suggestion
With Teleport, users can request access roles or to a specific set of resources.
Requests can be published to tools like Slack, Jira, and PagerDuty for easy integration with your organization's workflows.
```

> requesting specific resources such as a production server to allow fine-grain access for a targeted task.  The Web User Interface for Access Requests

Reviewer: The web interface for creating and approving access requests is only available in the Teleport Enterprise.
